deepmd/entrypoints/compress.py

```diff
@@ -64,7 +64,7 @@ def compress(
     """
     jdata = j_loader(INPUT)
     if "model" not in jdata.keys():
-        jdata = convert_input_v0_v1(jdata, warning=True, dump="input_v1_compat.json")
+        jdata = updata_deepmd_input(jdata, warning=True, dump="input_v1_compat.json")
```

Comment:
minor confusion: should default file name be `input_v2_compat.json`?

Reply:
Yes...